src/agents/supervisor/prompts.py

```diff
@@ -116,7 +116,9 @@
 Do not answer the user query yourself.
 
 First, decide whether to accept or reject the response.
-Accept the response if it answers the user query correctly and remove the agent names and your decision from your response.
+Accept the response:
+ - if it answers the user query correctly and remove the agent names and your decision from your response.
+ - if an agent cannot answer, mention it in the response.
```

Comment:
Will this info appear to the user in the final response?
We should avoid such cases!